pkg/aws/pca.go

```diff
@@ -181,6 +181,9 @@ func templateArn(caArn string, spec cmapi.CertificateRequestSpec) string {
 	arn := strings.SplitAfterN(caArn, ":", 3)
 	prefix := arn[0] + arn[1]
 
+	if spec.IsCA == true {
```

Comment:
Quick reminder - you also need to remove the check like I did at https://github.com/diranged/aws-privateca-issuer/commit/09c20605230f5d51e4f414321625172c9da7185e#diff-0023dd1178fb9031823ec9ecc0455654147cb6672a3170aa53c73dcc663e0054